libc-test/build.rs

```diff
@@ -2675,6 +2675,7 @@ fn test_linux(target: &str) {
                "elf.h",
                "fcntl.h",
                "glob.h",
+               "gnu/libc-version.h",
```

Comment:
It seems that the glibc versions we use in CI don't have this header file, which is causing the failure. Was this header added recently? If so then perhaps it would be better to add these symbols to the whitelist in libc-test/build.rs, with a note about which version of glibc they were introduced in.

Reply:
Ah I see the problem, it's failing on a musl target where this header doesn't exist. You need to prefix it with `[gnu]:`:
```
[gnu]: "gnu/libc-version.h",
```

Reply:
Ah cool, I'll get to this in a day or so, thanks for finding the issue!